armi/cases/case.py

```diff
@@ -716,31 +716,44 @@ def writeInputs(self, sourceDir: Optional[str] = None):
 
 
 def copyInputsHelper(
```

Comment:
Should this be `_copyInputsHelper()`, so that no one tries to use it outside of this file?

Your call, just wondering if it was meant to be public or not.

Reply:
I don't think it needs to be public, fixing